src/Illuminate/Foundation/Console/OptimizeClearCommand.php

```diff
@@ -31,15 +33,23 @@ public function handle()
     {
         $this->components->info('Clearing cached bootstrap files.');
 
-        collect([
-            'cache' => fn () => $this->callSilent('cache:clear') == 0,
-            'compiled' => fn () => $this->callSilent('clear-compiled') == 0,
-            'config' => fn () => $this->callSilent('config:clear') == 0,
-            'events' => fn () => $this->callSilent('event:clear') == 0,
-            'routes' => fn () => $this->callSilent('route:clear') == 0,
-            'views' => fn () => $this->callSilent('view:clear') == 0,
-        ])->each(fn ($task, $description) => $this->components->task($description, $task));
+        foreach ($this->getOptimizeClearTasks() as $description => $command) {
+            $this->components->task($description, fn () => $this->callSilently($command) == 0);
+        }
 
         $this->newLine();
     }
+
+    public function getOptimizeClearTasks(): array
+    {
+        return [
+            'cache' => 'cache:clear',
+            'compiled' => 'clear-compiled',
+            'config' => 'config:clear',
+            'events' => 'event:clear',
+            'routes' => 'route:clear',
+            'views' => 'view:clear',
```

Comment:
These could be registered in the respective service providers that are responsible for adding these commands/functionalities?